### PR TITLE
Add licenses of jquery-resize, cc #5500

### PR DESCRIPTION
**Pull request by yangwanting**
https://github.com/cowboy/jquery-resize/blob/master/LICENSE-GPL
https://github.com/cowboy/jquery-resize/blob/master/LICENSE-MIT
@x09326

### CODE REVIEW

File: ajax/libs/jquery-resize/package.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
       "jquery.ba-resize*.js"
     ]
   },
+  "licenses": [
+    "MIT",
+    "GPL-2.0"
+    ],
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-resize/package.json]:
```json
  "licenses": [
    "MIT",
    "GPL-2.0"
  ],
```

**Comment by PeterDaveHello** [comment]
@yangwanting your comment should be more meaningful and polite to ask for help/review from your mentor.
Reference: https://github.com/cdnjs/cdnjs/pull/9361

**Comment by x09326** [comment]
@cdnjs/intern2 Please help me review it.
Because she has quit, so I take over her work.

**Comment by x09326** [comment]
Close due to duplicated with #9869

**Comment by PeterDaveHello** [comment]
we close the old one if duplicated ... I don't think the reason is duplicated but she dropped...

**Pull request closed without merging**